src/node/abort.cpp

```diff
@@ -16,11 +16,11 @@
 
 namespace node {
 
-void AbortNode(util::SignalInterrupt* shutdown, std::atomic<int>& exit_status, const std::string& debug_message, const bilingual_str& user_message)
+void AbortNode(util::SignalInterrupt* shutdown, std::atomic<int>& exit_status, const bilingual_str& message)
 {
-    SetMiscWarning(Untranslated(debug_message));
-    LogPrintf("*** %s\n", debug_message);
-    InitError(user_message.empty() ? _("A fatal internal error occurred, see debug.log for details") : user_message);
+    SetMiscWarning(message);
+    LogPrintf("*** %s\n", message.original);
```

Comment:
```suggestion
    LogError("%s\n", message.original);
```

And potentially the same a few lines down, even though it's currently not touched.